src/components/Workflow/Workflow.css

```diff
@@ -2,8 +2,7 @@
 @import "./WorkflowCard.css";
 
 .nc-workflow {
-  padding: var(--pageMargin);
-  padding-bottom: 0;
+  padding: var(--pageMarginVertical) var(--pageMarginHorizontal) 0;
```

Comment:
@erquhart We could just do `padding: var(--pageMargin) 0;`, instead of splitting it into vertical and horizontal. I'm fine either way.

Reply:
Duh, totally forgot about the three value syntax, good point.